src/config.js

```diff
@@ -24,6 +24,7 @@ let config = {
   OS_WEB_URL: process.env.OS_WEB_URL || 'https://openstax.org',
   SEARCH_URL: process.env.SEARCH_URL || 'https://openstax.org',
   HIGHLIGHTS_URL: process.env.HIGHLIGHTS_URL || 'https://openstax.org',
+  ORIGIN_URL: typeof(window) === 'undefined' ? 'https://openstax.org' : window.location.origin,
```

Comment:
this would be probably `process.env.ORIGIN_URL || 'https://openstax.org'`
and in the dev config it could be `typeof(window) === 'undefined' ? 'https://openstax.org' : window.location.origin,` or 
`typeof(window) === 'undefined' ? 'https://openstax.org' : '',`